mir_dataset_loaders/utils.py

```diff
@@ -159,8 +182,7 @@ def download_from_remote(remote, data_home=None, clobber=False):
         os.path.join(MIR_DATASETS_DIR, remote.filename) if data_home is None
         else os.path.join(data_home, remote.filename)
     )
-
-    if not os.path.exists(download_path) or clobber:
+    if not os.path.exists(download_path) or clobber:  # TODO: @rabitt checking for the right file?
```

Comment:
If the file we're about to download already exists, we don't want to overwrite it.

Reply:
Ok, but we don't check if there's an unzipped version of the files already, right? so if we run `download` and there's no `.zip` file (because it was cleaned), we'll overwrite the unzipped files. Is that correct?